modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -237,31 +191,31 @@ SHA1 Fingerprint=3F:DA:BF:E7:A7:A7:90:62:CA:CF:C7:55:0E:1D:7D:05:16:7D:45:60
 $ {orch-cli} get pods -o=jsonpath='{.items[0].metadata.name}' -n __<workspace namespace>__ | grep '^workspace.*'
 ----
 
-- Get the name of the theia IDE container in the workspace POD with the following command:
+- Get the name of the Theia IDE container in the workspace POD with the following command:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get -o json pod __<workspace pod name>__  -n __<workspace namespace>__ | \
     jq -r '.spec.containers[] | select(.name | startswith("theia-ide")).name'
 ----
 
-- Look for a `ca-certs` ConfigMap that should have been created inside the workspace namespace:
+- Look for a `ca-certs` config map that should have been created inside the workspace namespace:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
 $ {orch-cli} get cm ca-certs __<workspace namespace>__
 ----
 
-- Check that the entries in the `ca-certs` config map contain all the additional entries you added in the certificate ConfigMap at the {prod-short} installation level, in addition to the the `ca-bundl.crt` entry which is a reserved one:
+- Check that the entries in the `ca-certs` config map contain all the additional entries you added before. In addition it may contain `ca-bundl.crt` entry which is a reserved one:
```

Comment:
I would keep `in the certificate ConfigMap at the {prod-short} installation level, in` but make it plural
`in the certificates ...`

Reply:
We can have several certificates in several config maps, so, in my opinion, avoiding specifying singular or plural here will be clearer. But I might be wrong.